cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -70,18 +62,18 @@
 
    settings:                An object used to customize the markup.
 
-   settings.heading:        A string containing heading text.
+   settings.heading:        A string containing heading text. ()
```

Comment:
Empty parentheses. 

FYI in other places optional parameters are put by the parameter itself, like https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/macros/notification.html#L11, https://github.com/cfpb/cfgov-refresh/blob/flapjack/cfgov/jinja2/v1/_includes/macros/time.html#L15, etc.